pkg/xkeyboard-config

```diff
@@ -1,10 +1,10 @@
 [mirrors]
-ftp://ftp.x.org/pub/individual/data/xkeyboard-config-1.4.tar.bz2
-http://ftp.x.org/pub/individual/data/xkeyboard-config-1.4.tar.bz2
+https://xorg.freedesktop.org/archive/individual/data/xkeyboard-config/xkeyboard-config-2.22.tar.bz2
```

Comment:
can you put at least one non-https link there ? 'cause busybox wget is not https capable without openssl binary